Iscra/Views/Habit/HabitCalender/HabitCalenderViewController.swift

```diff
@@ -8,21 +8,26 @@
 import UIKit
 import FSCalendar
 
-class HabitCalenderViewController: UIViewController  {
-    
+class HabitCalenderViewController: UIViewController {
     // MARK: Outlets
-    @IBOutlet weak var btnBack:UIButton!
-    @IBOutlet weak var lblTitle:UILabel!
-    @IBOutlet weak var btnShare:UIButton!
-    @IBOutlet weak var viewBottom:UIView!
-    @IBOutlet weak var lblDaysCount:UILabel!
-    @IBOutlet weak var btnEditHabit:UIButton!
-    @IBOutlet weak var btnBottomSheet:UIButton!
-    @IBOutlet weak var btnDeleteHabit:UIButton!
-    @IBOutlet weak var btnPreviousMonth:UIButton!
-    @IBOutlet var viewCalender:FSCalendar!
+    @IBOutlet weak var btnBack: UIButton!
```

Comment:
group all same type of controller e.g club all labels together same with button and son on @IBOutlet weak var lbl1: UILabel!
@IBOutlet weak var lbl2: UILabel!